scripts/md2html/build.sh

```diff
@@ -33,18 +30,18 @@ cp history/MAINTAINERS_v3.0.2.md history/MAINTAINERS_v3.0.3.md
 cp history/MAINTAINERS_v3.0.2.md history/MAINTAINERS_v3.1.0.md
 # add lines for 3.0.4, 3.1.1, ...
 
-cp -p js/* ../../deploy/js 2> /dev/null
-cp -p markdown/* ../../deploy/ 2> /dev/null
+cp -p js/* ../../docs/js 2> /dev/null
+cp -p markdown/* ../../docs/ 2> /dev/null
```

Comment:
This part of the script is executed in folder `scripts/md2html` of branch `main` which does not contain folders `js` or `markdown`. I think this is a left-over.

```suggestion
```

Reply:
Lines removed here https://github.com/OAI/OpenAPI-Specification/pull/3868/commits/9d361f65866229473ae8c94570cd48df2f152325